code/game/machinery/porta_turret/portable_turret_construct.dm

```diff
@@ -17,7 +18,30 @@
 	density = TRUE
 	var/build_step = PTURRET_UNSECURED //the current step in the building process
 	var/finish_name = "turret"	//the name applied to the product turret
+	var/obj/item/electronics/turret/electronics
 	var/obj/item/gun/installed_gun = null
+	
+/obj/machinery/porta_turret_construct/examine(mob/user)
```

Comment:
Could you add hints for the deconstruction of this as well? like in reinf_walls.dm